RealImpactAnalytics/FlaminGoal
# Implemented Firebase Google Login

**Changes Made**
- Implemented a temporary user-profile page to show profile details.
- Implemented AuthService to provide firebase login, and store user data in Firestore
- Implemented temporary updated on the app.component to login using google popup and logout
- Implemented AuthGuard to protect routes from unauthorized access. In this case user-profile page

**How to .test**
Run to the app, next to the giant super flamingo ball there is a tiny unstyled button waiting to be pressed.
You have to use your google account with the **company domain**. Otherwise, it won't work. (Configured that way through Google Cloud Console. In order to go public there is another process which required google's approval)
Now you can navigate /profile to see your details. PS: Scroll down.

@Carniatto I think I need help on ng unit testing.  They are broken
